lib/improver/ensemble_calibration/ensemble_calibration.py

```diff
@@ -995,142 +997,147 @@ def _spatial_domain_match(self):
 
         Returns:
-            (tuple): tuple containing:
-                **predicted_mean** (numpy.ndarray):
-                    Calibrated mean values in a flattened array.
-                **forecast_predictor** (iris.cube.Cube):
-                    The forecast predictors, mean values taken by collapsing
-                    the realization coordinate.
+            location_parameter (numpy.ndarray):
+                Location parameter calculated using the ensemble realizations
+                as the predictor.
         """
         forecast_predictor = self.current_forecast
 
-        # Calculate predicted mean = a + b*X, where X is the
-        # raw ensemble mean. In this case, b = beta^2.
+        # Calculate location parameter = a + b1*X1 .... + bn*Xn, where X is the
+        # ensemble realizations. The number of b and X terms depends upon the
+        # number of ensemble realizations. In this case, b = beta^2.
         beta_values = np.array([], dtype=np.float32)
         for key in optimised_coeffs.keys():
             if key.startswith("beta"):
                 beta_values = np.append(beta_values, optimised_coeffs[key])
         a_and_b = np.append(optimised_coeffs["alpha"], beta_values**2)
         forecast_predictor_flat = (
             convert_cube_data_to_2d(forecast_predictor))
-        forecast_var_flat = forecast_vars.data.flatten()
-        col_of_ones = np.ones(forecast_var_flat.shape, dtype=np.float32)
+        xy_shape = next(forecast_predictor.slices_over("realization")).shape
+        col_of_ones = np.ones(np.dot(*xy_shape), dtype=np.float32)
         ones_and_predictor = (
             np.column_stack((col_of_ones, forecast_predictor_flat)))
-        predicted_mean = np.dot(ones_and_predictor, a_and_b)
-        # Calculate mean of ensemble realizations, as only the
-        # calibrated ensemble mean will be returned.
-        forecast_predictor = (
-            forecast_predictor.collapsed(
-                "realization", iris.analysis.MEAN))
-
-        return predicted_mean, forecast_predictor
+        location_parameter = (
+            np.dot(ones_and_predictor, a_and_b).reshape(xy_shape).astype(
+                np.float32))
+        return location_parameter
 
-    @staticmethod
-    def calibrate_forecast_data(optimised_coeffs, predicted_mean,
-                                forecast_predictor, forecast_var):
+    def _calculate_scale_parameter(self, optimised_coeffs):
         """
-        Create a calibrated_forecast_predictor by reshaping the predicted mean
-        to the original domain dimensions. Apply the calibration coefficients
-        to the forecast data variance. Return both to give calibrated mean and
-        variance in the original domain dimensions.
+        Calculation of the scale parameter using the ensemble variance
+        adjusted using the gamma and delta coefficients calculated by EMOS.
+        This follows the equations below, where S^2 is the ensemble variance:
```

Comment:
I've updated this docstring to be a little clearer, and modified the indentation by including `.. math` prior to each equation, so that each equation is centred separately, rather than the centring  applying to all equations at once, which gives slightly different indentation.